x-pack/test/saved_object_tagging/api_integration/security_and_spaces/apis/create.ts

```diff
@@ -60,6 +60,7 @@ export default function ({ getService }: FtrProviderContext) {
         USERS.DEFAULT_SPACE_DASHBOARD_READ_USER,
         USERS.DEFAULT_SPACE_VISUALIZE_READ_USER,
         USERS.DEFAULT_SPACE_ADVANCED_SETTINGS_READ_USER,
+        USERS.DEFAULT_SPACE_MAPS_READ_USER,
```

Comment:
Could you please also add the user in the `delete`, `get`, `get_all`, `update` and `_find` suites in the same folder?